content/get-started/exploring-integrations/github-extensions-and-integrations.md

```diff
@@ -20,6 +20,12 @@ shortTitle: Extensions & integrations
 
 You can connect to {% data variables.product.product_name %} repositories within third-party editor tools such as Unity and {% data variables.product.prodname_vs %}.
 
+{% note %}
+
+**Note:** To see an example of how third-party editor tools connect to repositories for {% data variables.product.prodname_classroom %}, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/integrate-github-classroom-with-an-ide/integrate-github-classroom-with-an-ide)."
+
+{% endnote %}
+
```

Comment:
```suggestion
{% ifversion fpt %}
{% note %}

**Note:** To see an example of how third-party editor tools connect to repositories for {% data variables.product.prodname_classroom %}, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/integrate-github-classroom-with-an-ide/integrate-github-classroom-with-an-ide)."

{% endnote %}
{% endif %}
```

Suggestion to add versioning to fix broken links, same as for "Creating a template repository" (see my note there for an explanation of why these are "broken" 🙂)